lib/linguist/languages.yml

```diff
@@ -6492,6 +6504,8 @@ Tcl:
   - ".adp"
   - ".tcl.in"
   - ".tm"
+  - ".sdc"
+  - ".xdc"
```

Comment:
You'll need to alphabetise these, leaving the `.tcl` entry first as it is the primary extention.